packages/generator/README.md

```diff
@@ -0,0 +1,17 @@
+# About 
+Generator is a tool to quickly build new components without needing to setup configs, storybook, etc. 
+
+## Install
+Install Yeoman first: 
+`yarn global add yo`
+
+Then install this generator: 
+`yarn global add @kaizen/generator-create`
+
+## Usage
+`yo create` 
+`yo create:draft`
```

Comment:
I think these instructions are out of date:

These are the only commands that worked for me:

`yo` (and then choose @kaizen/create from available generators)
`yo @kaizen/create`

`yo @kaizen/create:draft` or `yo create:draft` did not exist

I think `@kaizen/create:draft` or `@kaizen/create-draft` is better than just `@kaizen/create` (what if we want to create legacy packages), but I'm not familiar with yeoman to know how to do this.

I tested this by using `yarn link`.

Reply:
They are, good catch!